test/v1/connectors/BaseDownstreamConnectorSpec.scala

```diff
@@ -50,6 +52,7 @@ class BaseDownstreamConnectorSpec extends ConnectorSpec {
     MockAppConfig.desToken returns "des-token"
     MockAppConfig.desEnvironment returns "des-environment"
     MockAppConfig.desEnvironmentHeaders returns desEnvironmentHeaders
+    MockAppConfig.featureSwitches returns Configuration("tys-api.enabled" -> false)
```

Comment:
Is it necessary to set tys-enabled to false? The tests pass without doing this. Also, the template in individuals-income-received api doesn't do this step either

Reply:
Verified tests do in fact pass without them. Removed and updated. Thanks!